make accept on linux still dies will c++ exception handling frame problems.

I can't seem to re-open https://github.com/cockroachdb/cockroach/issues/7477, but it is still a problem.

At tip:

```
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ git log|head
commit 7e2eb5bb615a5cfe1c8edbec9ff6bc8be18e8970
Merge: ffcdab5 2b204a2
Author: Peter Mattis <peter@cockroachlabs.com>
Date:   Fri Jul 8 11:05:16 2016 -0400

    Merge pull request #7696 from petermattis/pmattis/debug-keys
    
    cli: add `debug keys --sizes`

commit 2b204a2acc0f0d7737de34e38dfbb3bc353efb55
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ uname -a
Linux crdev 3.13.0-85-generic #129-Ubuntu SMP Thu Mar 17 20:50:15 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.4 LTS"
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ make acceptance
github.com/Sirupsen/logrus                         f3cfb454f4c2	[OK]
github.com/VividCortex/ewma                        c34099b489e4	[OK]
github.com/agtorre/gocolorize                      f42b554bf7f0	[OK]
github.com/biogo/store                             913427a1d5e8	[OK]
github.com/chzyer/readline                         683bf8ff7cd7	[OK]
github.com/client9/misspell                        6d3ee069a9d5	[OK]
github.com/cockroachdb/c-jemalloc                  9716a8692831	[OK]
github.com/cockroachdb/c-protobuf                  3bef67ea4cc8	[OK]
github.com/cockroachdb/c-rocksdb                   9dd1f18d3ab2	[OK]
github.com/cockroachdb/c-snappy                    d4e7b428fe7f	[OK]
github.com/cockroachdb/cmux                        9d8f8e431ac3	[OK]
github.com/cockroachdb/pq                          7c512ee21153	[OK]
github.com/cockroachdb/stress                      aa7690c22fd0	[OK]
github.com/codahale/hdrhistogram                   f8ad88b59a58	[OK]
github.com/coreos/etcd                             fb64c8ccfeb9	[OK]
github.com/cpuguy83/go-md2man                      2724a9c9051a	[OK]
github.com/docker/distribution                     ffbe9b7c6350	[OK]
github.com/docker/engine-api                       f90ecdb1e989	[OK]
github.com/docker/go-connections                   990a1a1a70b0	[OK]
github.com/docker/go-units                         f2d77a61e3c1	[OK]
github.com/dustin/go-humanize                      fef948f2d241	[OK]
github.com/elastic/gosigar                         f42b67097145	[checkout 61dfed4a9eb4]
github.com/elazarl/go-bindata-assetfs              57eb5e1fc594	[OK]
github.com/gengo/grpc-gateway                      04d03d2c0176	[OK]
github.com/go-sql-driver/mysql                     3654d25ec346	[OK]
github.com/gogo/protobuf                           318371cbef6b	[OK]
github.com/golang/glog                             23def4e6c14b	[OK]
github.com/golang/lint                             c7bacac2b21c	[OK]
github.com/golang/protobuf                         0c1f6d65b5a1	[OK]
github.com/google/btree                            7d79101e329e	[OK]
github.com/gordonklaus/ineffassign                 71e35c2a79cb	[OK]
github.com/jteeuwen/go-bindata                     a0ff2567cfb7	[OK]
github.com/julienschmidt/httprouter                77366a47451a	[OK]
github.com/kisielk/errcheck                        8e25ad9d46f6	[OK]
github.com/kisielk/gotool                          58a7a198f2ec	[OK]
github.com/kkaneda/returncheck                     bf081fa7155e	[OK]
github.com/kr/pretty                               add1dbc86daf	[OK]
github.com/kr/text                                 7cafcd837844	[OK]
github.com/lib/pq                                  e2402a7cd1e5	[OK]
github.com/lightstep/lightstep-tracer-go           45bde93e8d99	[OK]
github.com/mattn/go-isatty                         56b76bdf51f7	[OK]
github.com/mattn/go-runewidth                      d6bea18f7897	[OK]
github.com/mibk/dupl                               415e882ea21c	[OK]
github.com/montanaflynn/stats                      60dcacf48f43	[OK]
github.com/olekukonko/tablewriter                  daf2955e742c	[OK]
github.com/opencontainers/runc                     5dc3f3576efb	[OK]
github.com/opennota/check                          5b00aacd5639	[OK]
github.com/opentracing/basictracer-go              bc2e227fd86b	[OK]
github.com/opentracing/opentracing-go              6faad65c6060	[OK]
github.com/pkg/errors                              cbd18b544031	[OK]
github.com/rcrowley/go-metrics                     cfa5a85e9f0a	[OK]
github.com/robfig/glock                            e75a83a850fe	[OK]
github.com/russross/blackfriday                    1d6b8e9301e7	[OK]
github.com/satori/go.uuid                          879c5887cd47	[OK]
github.com/shurcooL/sanitized_anchor_name          10ef21a441db	[OK]
github.com/spf13/cobra                             6a8bd97bdb1f	[OK]
github.com/spf13/pflag                             367864438f1b	[OK]
github.com/stripe/safesql                          452e37ed7944	[OK]
github.com/tebeka/go2xunit                         22b8a642949a	[OK]
github.com/termie/go-shutil                        bcacb06fecae	[OK]
golang.org/x/crypto                                f3241ce85058	[OK]
golang.org/x/net                                   bc3663df0ac9	[OK]
golang.org/x/text                                  4440cd4f4c2e	[OK]
golang.org/x/tools                                 8b3828f1c8f7	[OK]
google.golang.org/grpc                             aecdccadd24d	[OK]
gopkg.in/inf.v0                                    3887ee99ecf0	[OK]
gopkg.in/yaml.v1                                   9f9df34309c0	[OK]
honnef.co/go/unused                                9057d575b3bb	[OK]
cmd github.com/client9/misspell/cmd/misspell                   	[OK]
cmd github.com/cockroachdb/c-protobuf/cmd/protoc               	[OK]
cmd github.com/cockroachdb/cockroach/cmd/protoc-gen-gogoroach  	[OK]
cmd github.com/cockroachdb/stress                              	[OK]
cmd github.com/gengo/grpc-gateway/protoc-gen-grpc-gateway      	[OK]
cmd github.com/golang/lint/golint                              	[OK]
cmd github.com/gordonklaus/ineffassign                         	[OK]
cmd github.com/jteeuwen/go-bindata/go-bindata                  	[OK]
cmd github.com/kisielk/errcheck                                	[OK]
cmd github.com/kkaneda/returncheck                             	[OK]
cmd github.com/mibk/dupl                                       	[OK]
cmd github.com/opennota/check/cmd/varcheck                     	[OK]
cmd github.com/robfig/glock                                    	[OK]
cmd github.com/stripe/safesql                                  	[OK]
cmd github.com/tebeka/go2xunit                                 	[OK]
cmd golang.org/x/tools/cmd/goimports                           	[OK]
cmd golang.org/x/tools/cmd/stringer                            	[OK]
cmd honnef.co/go/unused/cmd/unused                             	[OK]
touch .bootstrap
GOPATH set to /go
/go/bin added to PATH
go install -v -tags clockoffset
github.com/cockroachdb/cockroach/cli/cliflags
github.com/cockroachdb/cockroach/util
github.com/cockroachdb/cockroach/base
github.com/cockroachdb/cockroach/util/encoding
github.com/cockroachdb/cockroach/gossip/resolver
github.com/cockroachdb/cockroach/roachpb
github.com/elastic/gosigar
github.com/cockroachdb/cockroach/storage/engine/rocksdb
github.com/cockroachdb/cockroach/keys
github.com/cockroachdb/cockroach/util/stop
github.com/cockroachdb/cockroach/config
github.com/cockroachdb/cockroach/util/netutil
github.com/cockroachdb/cockroach/util/grpcutil
github.com/cockroachdb/cockroach/rpc
github.com/cockroachdb/cockroach/internal/client
github.com/cockroachdb/cockroach/gossip
github.com/cockroachdb/cockroach/kv
github.com/cockroachdb/cockroach/ts/tspb
github.com/cockroachdb/cockroach/server/status
github.com/cockroachdb/cockroach/storage/storagebase
github.com/cockroachdb/cockroach/server/serverpb
github.com/cockroachdb/cockroach/server/testingshim
github.com/cockroachdb/cockroach/sql/parser
github.com/cockroachdb/cockroach/sql/sqlbase
github.com/cockroachdb/cockroach/sql/distsql
github.com/cockroachdb/cockroach/sql/sqlutil
github.com/cockroachdb/cockroach/sql
github.com/cockroachdb/cockroach/storage/engine
github.com/cockroachdb/cockroach/storage
github.com/cockroachdb/cockroach/sql/pgwire
github.com/cockroachdb/cockroach/ts
github.com/cockroachdb/cockroach/ui
github.com/cockroachdb/cockroach/server
github.com/cockroachdb/cockroach/cli
github.com/cockroachdb/cockroach
# github.com/cockroachdb/cockroach
/usr/local/go/pkg/tool/linux_amd64/link: running g++ failed: exit status 1
/usr/bin/ld: .eh_frame_hdr table[5308] FDE at 000000000231a818 overlaps table[5309] FDE at 000000000231af08.
/usr/bin/ld: .eh_frame_hdr table[5309] FDE at 000000000231af08 overlaps table[5310] FDE at 000000000231f4d0.
/usr/bin/ld: .eh_frame_hdr table[5310] FDE at 000000000231f4d0 overlaps table[5311] FDE at 0000000002324c30.
/usr/bin/ld: .eh_frame_hdr table[5311] FDE at 0000000002324c30 overlaps table[5312] FDE at 000000000232f7c8.
/usr/bin/ld: .eh_frame_hdr table[5312] FDE at 000000000232f7c8 overlaps table[5313] FDE at 000000000232fda8.
/usr/bin/ld: .eh_frame_hdr table[5313] FDE at 000000000232fda8 overlaps table[5314] FDE at 0000000002330320.
/usr/bin/ld: .eh_frame_hdr table[5314] FDE at 0000000002330320 overlaps table[5315] FDE at 0000000002333868.
/usr/bin/ld: .eh_frame_hdr table[5315] FDE at 0000000002333868 overlaps table[5316] FDE at 0000000002333c58.
/usr/bin/ld: .eh_frame_hdr table[5316] FDE at 0000000002333c58 overlaps table[5317] FDE at 0000000002334390.
/usr/bin/ld: .eh_frame_hdr table[5317] FDE at 0000000002334390 overlaps table[5318] FDE at 0000000002334e10.
/usr/bin/ld: .eh_frame_hdr table[5318] FDE at 0000000002334e10 overlaps table[5319] FDE at 000000000233b7a8.
...
/usr/bin/ld: .eh_frame_hdr table[11966] FDE at 0000000002388858 overlaps table[11967] FDE at 000000000238aa38.
/usr/bin/ld: .eh_frame_hdr table[11968] FDE at 0000000002388890 overlaps table[11969] FDE at 000000000238aa00.
/usr/bin/ld: .eh_frame_hdr table[11970] FDE at 00000000023888c8 overlaps table[11971] FDE at 000000000238aa98.
/usr/bin/ld: .eh_frame_hdr table[11980] FDE at 0000000002389420 overlaps table[11981] FDE at 000000000238ac60.
/usr/bin/ld: .eh_frame_hdr table[11982] FDE at 0000000002389450 overlaps table[11983] FDE at 000000000238ad00.
/usr/bin/ld: .eh_frame_hdr table[11984] FDE at 0000000002389488 overlaps table[11985] FDE at 000000000238ad38.
/usr/bin/ld: .eh_frame_hdr table[11987] FDE at 00000000023894f0 overlaps table[11988] FDE at 000000000238ade8.
/usr/bin/ld: .eh_frame_hdr table[11992] FDE at 00000000023896a0 overlaps table[11993] FDE at 000000000238ae18.
/usr/bin/ld: .eh_frame_hdr table[11995] FDE at 00000000023898c8 overlaps table[11996] FDE at 000000000238a8b8.
collect2: error: ld returned 1 exit status

Makefile:80: recipe for target 'install' failed
make: *** [install] Error 2
make: *** [acceptance] Error 2
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ git log|head
commit 7e2eb5bb615a5cfe1c8edbec9ff6bc8be18e8970
Merge: ffcdab5 2b204a2
Author: Peter Mattis <peter@cockroachlabs.com>
Date:   Fri Jul 8 11:05:16 2016 -0400

    Merge pull request #7696 from petermattis/pmattis/debug-keys
    
    cli: add `debug keys --sizes`

commit 2b204a2acc0f0d7737de34e38dfbb3bc353efb55
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ git reset --hard master
HEAD is now at 7e2eb5b Merge pull request #7696 from petermattis/pmattis/debug-keys
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ make acceptance
GOPATH set to /go
/go/bin added to PATH
go install -v -tags clockoffset
github.com/cockroachdb/cockroach
# github.com/cockroachdb/cockroach
/usr/local/go/pkg/tool/linux_amd64/link: running g++ failed: exit status 1
/usr/bin/ld: .eh_frame_hdr table[5308] FDE at 000000000231a818 overlaps table[5309] FDE at 000000000231af08.
/usr/bin/ld: .eh_frame_hdr table[5309] FDE at 000000000231af08 overlaps table[5310] FDE at 000000000231f4d0.
/usr/bin/ld: .eh_frame_hdr table[5310] FDE at 000000000231f4d0 overlaps table[5311] FDE at 0000000002324c30.
/usr/bin/ld: .eh_frame_hdr table[5311] FDE at 0000000002324c30 overlaps table[5312] FDE at 000000000232f7c8.
/usr/bin/ld: .eh_frame_hdr table[5312] FDE at 000000000232f7c8 overlaps table[5313] FDE at 000000000232fda8.
...
/usr/bin/ld: .eh_frame_hdr table[11968] FDE at 0000000002388890 overlaps table[11969] FDE at 000000000238aa00.
/usr/bin/ld: .eh_frame_hdr table[11970] FDE at 00000000023888c8 overlaps table[11971] FDE at 000000000238aa98.
/usr/bin/ld: .eh_frame_hdr table[11980] FDE at 0000000002389420 overlaps table[11981] FDE at 000000000238ac60.
/usr/bin/ld: .eh_frame_hdr table[11982] FDE at 0000000002389450 overlaps table[11983] FDE at 000000000238ad00.
/usr/bin/ld: .eh_frame_hdr table[11984] FDE at 0000000002389488 overlaps table[11985] FDE at 000000000238ad38.
/usr/bin/ld: .eh_frame_hdr table[11987] FDE at 00000000023894f0 overlaps table[11988] FDE at 000000000238ade8.
/usr/bin/ld: .eh_frame_hdr table[11992] FDE at 00000000023896a0 overlaps table[11993] FDE at 000000000238ae18.
/usr/bin/ld: .eh_frame_hdr table[11995] FDE at 00000000023898c8 overlaps table[11996] FDE at 000000000238a8b8.
collect2: error: ld returned 1 exit status

Makefile:80: recipe for target 'install' failed
make: *** [install] Error 2
make: *** [acceptance] Error 2
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ 

```

No dice. It's still a problem, even after cleaning out bin/ and pkg/

```
jaten@crdev:~/go/bin$ cd ..
jaten@crdev:~/go$ ls
bin  cockroach-data  pkg  src
jaten@crdev:~/go$ rm -rf bin
jaten@crdev:~/go$ mkdir bin
jaten@crdev:~/go$ cd pkg
jaten@crdev:~/go/pkg$ ls
docker_amd64  docker_amd64_race  linux_amd64  linux_amd64_race
jaten@crdev:~/go/pkg$ rm -rf *
rm: cannot remove ‘linux_amd64/github.com/cockroachdb/cockroach/internal/client.a’: Permission denied
jaten@crdev:~/go/pkg$ sudo su
root@crdev:/home/jaten/go/pkg# rm -rf *
root@crdev:/home/jaten/go/pkg# ls
root@crdev:/home/jaten/go/pkg# exit
jaten@crdev:~/go/pkg$ ls
jaten@crdev:~/go/pkg$ cd ..
jaten@crdev:~/go$ ls
bin  cockroach-data  pkg  src
jaten@crdev:~/go$ cd src
jaten@crdev:~/go/src$ ls
github.com  golang.org	google.golang.org  gopkg.in  honnef.co
jaten@crdev:~/go/src$ cd github.com/
jaten@crdev:~/go/src/github.com$ ls
agtorre      cpuguy83  gogo	      kisielk	 montanaflynn	 robfig    syndtr
biogo	     docker    golang	      kkaneda	 olekukonko	 russross  tebeka
chzyer	     dustin    google	      kr	 opencontainers  satori    termie
client9      elastic   gordonklaus    lib	 opennota	 shurcooL  urfave
cockroachdb  elazarl   go-sql-driver  lightstep  opentracing	 Sirupsen  vishvananda
codahale     gengo     jteeuwen       mattn	 pkg		 spf13	   VividCortex
coreos	     godbus    julienschmidt  mibk	 rcrowley	 stripe
jaten@crdev:~/go/src/github.com$ cd cockroachdb/
jaten@crdev:~/go/src/github.com/cockroachdb$ ls
c-jemalloc  cmux  cockroach  c-protobuf  c-rocksdb  c-snappy  pq  stress
jaten@crdev:~/go/src/github.com/cockroachdb$ cd cockroach/
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ ls
acceptance  cli     CONTRIBUTING.md  gossip    LICENSE	  resource  server    testutils
base	    cloud   docs	     internal  main.go	  roachpb   sql       ts
build	    cmd     githooks	     keys      Makefile   rpc	    storage   ui
circle.yml  config  GLOCKFILE	     kv        README.md  security  STYLE.md  util
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ make acceptance
go get github.com/robfig/glock
github.com/Sirupsen/logrus                         f3cfb454f4c2	[OK]
github.com/VividCortex/ewma                        c34099b489e4	[OK]
github.com/agtorre/gocolorize                      f42b554bf7f0	[OK]
github.com/biogo/store                             913427a1d5e8	[OK]
github.com/chzyer/readline                         683bf8ff7cd7	[OK]
github.com/client9/misspell                        6d3ee069a9d5	[OK]
github.com/cockroachdb/c-jemalloc                  9716a8692831	[OK]
github.com/cockroachdb/c-protobuf                  3bef67ea4cc8	[OK]
github.com/cockroachdb/c-rocksdb                   9dd1f18d3ab2	[OK]
github.com/cockroachdb/c-snappy                    d4e7b428fe7f	[OK]
github.com/cockroachdb/cmux                        9d8f8e431ac3	[OK]
github.com/cockroachdb/pq                          7c512ee21153	[OK]
github.com/cockroachdb/stress                      aa7690c22fd0	[OK]
github.com/codahale/hdrhistogram                   f8ad88b59a58	[OK]
github.com/coreos/etcd                             fb64c8ccfeb9	[OK]
github.com/cpuguy83/go-md2man                      2724a9c9051a	[OK]
github.com/docker/distribution                     ffbe9b7c6350	[OK]
github.com/docker/engine-api                       f90ecdb1e989	[OK]
github.com/docker/go-connections                   990a1a1a70b0	[OK]
github.com/docker/go-units                         f2d77a61e3c1	[OK]
github.com/dustin/go-humanize                      fef948f2d241	[OK]
github.com/elastic/gosigar                         61dfed4a9eb4	[OK]
github.com/elazarl/go-bindata-assetfs              57eb5e1fc594	[OK]
github.com/gengo/grpc-gateway                      04d03d2c0176	[OK]
github.com/go-sql-driver/mysql                     3654d25ec346	[OK]
github.com/gogo/protobuf                           318371cbef6b	[OK]
github.com/golang/glog                             23def4e6c14b	[OK]
github.com/golang/lint                             c7bacac2b21c	[OK]
github.com/golang/protobuf                         0c1f6d65b5a1	[OK]
github.com/google/btree                            7d79101e329e	[OK]
github.com/gordonklaus/ineffassign                 71e35c2a79cb	[OK]
github.com/jteeuwen/go-bindata                     a0ff2567cfb7	[OK]
github.com/julienschmidt/httprouter                77366a47451a	[OK]
github.com/kisielk/errcheck                        8e25ad9d46f6	[OK]
github.com/kisielk/gotool                          58a7a198f2ec	[OK]
github.com/kkaneda/returncheck                     bf081fa7155e	[OK]
github.com/kr/pretty                               add1dbc86daf	[OK]
github.com/kr/text                                 7cafcd837844	[OK]
github.com/lib/pq                                  e2402a7cd1e5	[OK]
github.com/lightstep/lightstep-tracer-go           45bde93e8d99	[OK]
github.com/mattn/go-isatty                         56b76bdf51f7	[OK]
github.com/mattn/go-runewidth                      d6bea18f7897	[OK]
github.com/mibk/dupl                               415e882ea21c	[OK]
github.com/montanaflynn/stats                      60dcacf48f43	[OK]
github.com/olekukonko/tablewriter                  daf2955e742c	[OK]
github.com/opencontainers/runc                     5dc3f3576efb	[OK]
github.com/opennota/check                          5b00aacd5639	[OK]
github.com/opentracing/basictracer-go              bc2e227fd86b	[OK]
github.com/opentracing/opentracing-go              6faad65c6060	[OK]
github.com/pkg/errors                              cbd18b544031	[OK]
github.com/rcrowley/go-metrics                     cfa5a85e9f0a	[OK]
github.com/robfig/glock                            e75a83a850fe	[OK]
github.com/russross/blackfriday                    1d6b8e9301e7	[OK]
github.com/satori/go.uuid                          879c5887cd47	[OK]
github.com/shurcooL/sanitized_anchor_name          10ef21a441db	[OK]
github.com/spf13/cobra                             6a8bd97bdb1f	[OK]
github.com/spf13/pflag                             367864438f1b	[OK]
github.com/stripe/safesql                          452e37ed7944	[OK]
github.com/tebeka/go2xunit                         22b8a642949a	[OK]
github.com/termie/go-shutil                        bcacb06fecae	[OK]
golang.org/x/crypto                                f3241ce85058	[OK]
golang.org/x/net                                   bc3663df0ac9	[OK]
golang.org/x/text                                  4440cd4f4c2e	[OK]
golang.org/x/tools                                 8b3828f1c8f7	[OK]
google.golang.org/grpc                             aecdccadd24d	[OK]
gopkg.in/inf.v0                                    3887ee99ecf0	[OK]
gopkg.in/yaml.v1                                   9f9df34309c0	[OK]
honnef.co/go/unused                                9057d575b3bb	[OK]
cmd github.com/client9/misspell/cmd/misspell                   	[built]
cmd github.com/cockroachdb/c-protobuf/cmd/protoc               	[built]
cmd github.com/cockroachdb/cockroach/cmd/protoc-gen-gogoroach  	[built]
cmd github.com/cockroachdb/stress                              	[built]
cmd github.com/gengo/grpc-gateway/protoc-gen-grpc-gateway      	[built]
cmd github.com/golang/lint/golint                              	[built]
cmd github.com/gordonklaus/ineffassign                         	[built]
cmd github.com/jteeuwen/go-bindata/go-bindata                  	[built]
cmd github.com/kisielk/errcheck                                	[built]
cmd github.com/kkaneda/returncheck                             	[built]
cmd github.com/mibk/dupl                                       	[built]
cmd github.com/opennota/check/cmd/varcheck                     	[built]
cmd github.com/robfig/glock                                    	[OK]
cmd github.com/stripe/safesql                                  	[built]
cmd github.com/tebeka/go2xunit                                 	[built]
cmd golang.org/x/tools/cmd/goimports                           	[built]
cmd golang.org/x/tools/cmd/stringer                            	[built]
cmd honnef.co/go/unused/cmd/unused                             	[built]
touch .bootstrap
GOPATH set to /go
/go/bin added to PATH
go install -v -tags clockoffset
golang.org/x/crypto/ssh/terminal
github.com/cockroachdb/c-jemalloc
github.com/chzyer/readline
github.com/cockroachdb/cockroach/cli/cliflags
github.com/dustin/go-humanize
github.com/cockroachdb/cockroach/util/humanizeutil
github.com/cockroachdb/cockroach/build
github.com/cockroachdb/cockroach/util/caller
github.com/cockroachdb/cockroach/util/log/logflags
golang.org/x/net/context
github.com/opentracing/opentracing-go
github.com/cockroachdb/cockroach/util/log
github.com/cockroachdb/cockroach/util/envutil
github.com/cockroachdb/cockroach/util/timeutil
github.com/pkg/errors
golang.org/x/crypto/blowfish
golang.org/x/crypto/bcrypt
github.com/cockroachdb/cockroach/security
github.com/cockroachdb/cockroach/util/retry
github.com/cockroachdb/cockroach/base
github.com/biogo/store/llrb
github.com/biogo/store/interval
github.com/cockroachdb/cockroach/util/hlc
github.com/satori/go.uuid
github.com/cockroachdb/cockroach/util/uuid
github.com/cockroachdb/cockroach/storage/engine/enginepb
github.com/cockroachdb/cockroach/util/protoutil
github.com/gengo/grpc-gateway/runtime/internal
github.com/golang/protobuf/jsonpb
golang.org/x/net/http2/hpack
golang.org/x/net/lex/httplex
golang.org/x/net/http2
golang.org/x/net/internal/timeseries
golang.org/x/net/trace
google.golang.org/grpc/codes
google.golang.org/grpc/credentials
google.golang.org/grpc/grpclog
google.golang.org/grpc/internal
google.golang.org/grpc/metadata
google.golang.org/grpc/naming
google.golang.org/grpc/peer
google.golang.org/grpc/transport
google.golang.org/grpc
github.com/gengo/grpc-gateway/runtime
github.com/gogo/protobuf/jsonpb
github.com/cockroachdb/cockroach/util
github.com/cockroachdb/cockroach/util/duration
gopkg.in/inf.v0
github.com/cockroachdb/cockroach/util/decimal
github.com/cockroachdb/cockroach/util/encoding
github.com/gogo/protobuf/sortkeys
github.com/lightstep/lightstep-tracer-go/thrift_0_9_2/lib/go/thrift
github.com/lightstep/lightstep-tracer-go/lightstep_thrift
github.com/lightstep/lightstep-tracer-go/truncator
github.com/opentracing/basictracer-go/wire
github.com/opentracing/opentracing-go/ext
github.com/opentracing/basictracer-go
github.com/lightstep/lightstep-tracer-go
github.com/opentracing/basictracer-go/events
github.com/cockroachdb/cockroach/util/tracing
github.com/cockroachdb/cockroach/roachpb
github.com/cockroachdb/cmux
github.com/VividCortex/ewma
github.com/codahale/hdrhistogram
github.com/rcrowley/go-metrics
github.com/cockroachdb/cockroach/util/metric
github.com/cockroachdb/cockroach/gossip/resolver
github.com/cockroachdb/cockroach/util/interval
github.com/cockroachdb/cockroach/util/cache
github.com/elastic/gosigar
github.com/cockroachdb/cockroach/keys
github.com/cockroachdb/cockroach/util/stop
github.com/cockroachdb/cockroach/util/netutil
github.com/cockroachdb/cockroach/config
github.com/cockroachdb/cockroach/util/grpcutil
github.com/cockroachdb/cockroach/rpc
github.com/cockroachdb/cockroach/ts/tspb
github.com/cockroachdb/cockroach/internal/client
github.com/cockroachdb/cockroach/gossip
github.com/cockroachdb/cockroach/server/status
github.com/cockroachdb/cockroach/kv
github.com/cockroachdb/cockroach/storage/storagebase
github.com/cockroachdb/cockroach/server/serverpb
github.com/cockroachdb/cockroach/server/testingshim
github.com/cockroachdb/cockroach/sql/privilege
github.com/cockroachdb/cockroach/sql/parser
github.com/cockroachdb/cockroach/sql/pgwire/pgerror
golang.org/x/text/transform
golang.org/x/text/unicode/norm
github.com/cockroachdb/cockroach/sql/sqlutil
github.com/lib/pq/oid
github.com/cockroachdb/pq
github.com/cockroachdb/pq/oid
github.com/cockroachdb/c-snappy
github.com/cockroachdb/cockroach/sql/sqlbase
github.com/cockroachdb/c-rocksdb
github.com/cockroachdb/cockroach/sql/distsql
github.com/cockroachdb/cockroach/sql
github.com/cockroachdb/cockroach/sql/pgwire
github.com/cockroachdb/cockroach/util/bufalloc
github.com/coreos/etcd/raft/raftpb
github.com/coreos/etcd/raft
github.com/google/btree
github.com/kr/text
github.com/kr/pretty
github.com/cockroachdb/cockroach/ts
github.com/cockroachdb/cockroach/ui
github.com/cockroachdb/cockroach/util/sdnotify
github.com/elazarl/go-bindata-assetfs
github.com/julienschmidt/httprouter
github.com/rcrowley/go-metrics/exp
github.com/mattn/go-isatty
github.com/mattn/go-runewidth
github.com/olekukonko/tablewriter
github.com/spf13/pflag
github.com/spf13/cobra
github.com/shurcooL/sanitized_anchor_name
github.com/russross/blackfriday
github.com/cpuguy83/go-md2man/md2man
github.com/spf13/cobra/doc
gopkg.in/yaml.v1
github.com/cockroachdb/cockroach/util/randutil
github.com/cockroachdb/cockroach/storage/engine/rocksdb
github.com/cockroachdb/cockroach/storage/engine
github.com/cockroachdb/cockroach/storage
github.com/cockroachdb/cockroach/server
github.com/cockroachdb/cockroach/cli
github.com/cockroachdb/cockroach
# github.com/cockroachdb/cockroach
/usr/local/go/pkg/tool/linux_amd64/link: running g++ failed: exit status 1
/usr/bin/ld: .eh_frame_hdr table[5307] FDE at 0000000002315bb0 overlaps table[5308] FDE at 00000000023162a0.
/usr/bin/ld: .eh_frame_hdr table[5308] FDE at 00000000023162a0 overlaps table[5309] FDE at 000000000231a890.
/usr/bin/ld: .eh_frame_hdr table[5309] FDE at 000000000231a890 overlaps table[5310] FDE at 000000000231ff70.
/usr/bin/ld: .eh_frame_hdr table[5310] FDE at 000000000231ff70 overlaps table[5311] FDE at 000000000232ab40.
/usr/bin/ld: .eh_frame_hdr table[5311] FDE at 000000000232ab40 overlaps table[5312] FDE at 000000000232b128.
...
/usr/bin/ld: .eh_frame_hdr table[6424] FDE at 0000000002340140 overlaps table[6425] FDE at 0000000002341680.
/usr/bin/ld: .eh_frame_hdr table[6650] FDE at 0000000002349ab8 overlaps table[6651] FDE at 000000000234c238.
/usr/bin/ld: .eh_frame_hdr table[6652] FDE at 0000000002349ae8 overlaps table[6653] FDE at 000000000234c268.
collect2: error: ld returned 1 exit status

Makefile:80: recipe for target 'install' failed
make: *** [install] Error 2
make: *** [acceptance] Error 2
jaten@crdev:~/go/src/github.com/cockroachdb/cockroach$ 
```

--- a/build/builder.sh
+++ b/build/builder.sh
@@ -66,21 +66,22 @@ cockroach_toplevel="$(dirname $(cd $(dirname $0); pwd))"
 # This script supports both circleci and development hosts, so it must
 # support cases where the architecture inside the container is
 # different from that outside the container. We map /src/ directly
-# into the container because it is architecture-independent, and /pkg/
-# because every subdirectory is tagged with its architecture. We also
-# map certain subdirectories of ${GOPATH}/pkg into ${GOROOT}/pkg so
-# they can be used to cache builds of the standard library. /bin/ is
-# mapped separately to avoid clobbering the host's binaries. Note that
-# the path used for the /bin/ mapping is also used in the defaultBinary
+# into the container because it is architecture-independent. We then
+# map certain subdirectories of ${GOPATH}/pkg into both ${GOPATH}/pkg
+# and ${GOROOT}/pkg. The ${GOROOT} mapping is needed so they can be
+# used to cache builds of the standard library. /bin/ is mapped
+# separately to avoid clobbering the host's binaries. Note that the
+# path used for the /bin/ mapping is also used in the defaultBinary
 # function of localcluster.go.
 #
 # -i causes some commands (including `git diff`) to attempt to use
 # a pager, so we override $PAGER to disable.
 docker run -i ${tty-} ${rm} \
   --volume="${gopath0}/src:/go/src" \
-  --volume="${gopath0}/pkg:/go/pkg" \
-  --volume="${gopath0}/pkg/docker_amd64:/usr/src/go/pkg/linux_amd64" \
-  --volume="${gopath0}/pkg/docker_amd64_race:/usr/src/go/pkg/linux_amd64_race" \
+  --volume="${gopath0}/pkg/docker_amd64:/go/pkg/linux_amd64" \
+  --volume="${gopath0}/pkg/docker_amd64_race:/go/pkg/linux_amd64_race" \
+  --volume="${gopath0}/pkg/docker_amd64:/usr/local/go/pkg/linux_amd64" \
+  --volume="${gopath0}/pkg/docker_amd64_race:/usr/local/go/pkg/linux_amd64_race" \
   --volume="${gopath0}/bin/docker_amd64:/go/bin" \
   --volume="${HOME}/.jspm:/root/.jspm" \
   --volume="${HOME}/.npm:/root/.npm" \

